### ISSUE TITLE
Doesn't work with ES6 class syntax

### ISSUE THREAD

**Issue by darekrossman** [alexlande/react-to-jsx#10]
Great little utility here! One thing I'm having issues with is using it with components created with the ES6 class syntax. It ends up printing something like this:

``` javascript
<function Button(props) { _classCallCheck(this, Button); _get(Object.getPrototypeOf(Button.prototype), "constructor", this).call(this, props); this.state = { hovered: false, active: false }; }> Default </function Button(props) { _classCallCheck(this, Button); _get(Object.getPrototypeOf(Button.prototype), "constructor", this).call(this, props); this.state = { hovered: false, active: false }; }>
```

I'm still trying to wrap my head around the React internals so I'll be digging into this myself - but was hoping maybe you had an idea of what was happening?

**Comment by alexlande** [comment]
Weird! I sort of figured that would just work but never tried it. I'll dig in and see what I can find.

**Comment by darekrossman** [comment]
I think I see what's happening. component.type.displayName is actually component.type.name for the class-style components. Just had to change this bit in componentToJson()

```
  if (component.type && (component.type.displayName || component.type.name)) {
    name = component.type.displayName || component.type.name;
  } else if (component.type) {
    name = component.type;
  } else {
    return component;
  }
```

**Comment by alexlande** [comment]
Oh, that's easy enough. Feel free to submit a PR, otherwise I can handle it tonight. Thanks!

**Comment by subpopular** [comment]
#11

**Comment by alexlande** [comment]
Just published 1.3.1 with this fix. Cheers!
